data/star_data/zams_models/create_z2m2_y28/src/run_star_extras.f90

```diff
@@ -379,7 +379,7 @@ integer function evolve_to_zams_check_model(s, id, lipar, ipar, lrpar, rpar)
          real(dp), intent(inout), pointer :: rpar(:) ! (lrpar)
          evolve_to_zams_check_model = bare_bones_check_model(id) 
          if (evolve_to_zams_check_model /= keep_going) return
-         if (s% L_nuc_burn_total >= s% L(1)/Lsun) then
+         if (s% L_nuc_burn_total >= 0.999d0 * s% L(1)/Lsun) then
```

Comment:
Is this basically the same as the [control used](https://docs.mesastar.org/en/release-r23.05.1/reference/controls.html#lnuc-div-l-zams-limit) in the work folder?

https://github.com/MESAHub/mesa/blob/ee3de3fd1f0c26f3abe9344becf4cde0e5a39cc6/star/work/inlist_project#L48-L50

If so, could we use that control, in case people want to generate their own MS tables without having to modify this `run_star_extras.f90`?

Reply:
I believe it ends up being identical to how stopping near ZAMS works in inlist controls. Since it's the same for all masses, we can probably move it into the inlist, but given the non-standard setup in this directory, that might have ramifications I haven't thought through.

I'll give it a try.

Reply:
I tried this, but the control is not respected. I think it's because this part of the evolution is triggered by the `evolve_to_check_point`, which can only be stopped by `extras_check_model` (or so the comment in `star_lib.f90` seems to indicate). So for now, this needs to just live in `run_star_extras.f90`.